build/inc/FLoC/Component.php

```diff
@@ -59,18 +59,18 @@ public function initialize_floc_blocking() {
 	public function blocking_methods( $methods = array() ) {
 
 		$methods['none'] = array(
-			'title'       => __( 'None / Allow FLoC' ),
+			'title'       => __( 'None / Allow FLoC', 'wpm-floc' ),
 			'description' => __( 'This method does not provide an opt-out from FLoC.', 'wpm-floc' ),
 		);
 
 		$methods['simple'] = array(
 			'title'       => __( 'Simple / PHP' ),
-			'callback'    => array( $this, 'initialize_simple' ),
+			'callback'    => array( $this, 'initialize_simple', 'wpm-floc' ),
```

Comment:
I am not the maintainer and not sure if/when/or how the PR will get merged. Just browsed by out of interest and saw this; Seems, you did choose the wrong line for the text domain. Should have been added in L67 imo.